src/services/helper/sessions.js

```diff
@@ -12,6 +12,7 @@ const kafkaCommunication = require('@generics/kafka-communication')
 const apiBaseUrl = process.env.USER_SERIVCE_HOST + process.env.USER_SERIVCE_BASE_URL
 const request = require('request')
 
+const axios = require('axios')
```

Comment:
@nevil-mathew why this module, are we using this ?

Reply:
no, couldn't find it.